### DISCUSSION TITLE
Class 'Illuminate\Suppourt\Facades\DB' not found

### DISCUSSION THREAD

**Discussion by NesmaJarour** [laravel/framework#33186] [General] [answered]
Hello!
He has a problem here ..
  Class 'Illuminate\Suppourt\Facades\DB' not found

  at C:\xampp\htdocs\laravelblog\database\seeds\CategoriesTableSeeder.php:17
    13|      * @return void
    14|      */
    15|     public function run()
    16|     {
  > 17|       DB::table('categories')->insert([
    18|         [
    19|         'user_id' => '1',
    20|         'name' => 'Laravel',
    21|         'slug' => 'laravel',

  1   C:\xampp\htdocs\laravelblog\vendor\laravel\framework\src\Illuminate\Container\BoundMethod.php:33
      CategoriesTableSeeder::run()
  2   C:\xampp\htdocs\laravelblog\vendor\laravel\framework\src\Illuminate\Container\BoundMethod.php:33
      call_user_func_array([])

C:\xampp\htdocs\laravelblog>

**Comment by GrahamCampbell** [comment, marked as answer]
You have spelt `Support` wrong. :)

**Comment by NesmaJarour** [comment]
Thanks! Problem solved ... but a new problem arose :)
When I write 
C:\xampp\htdocs\laravelblog>php artisan db:seed --class=CategoriesTableSeeder

   Illuminate\Database\QueryException

  SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'Laravel' for key 'categories_name_unique' (SQL: insert into `categories` (`created_at`, `is_published`, `name`, `slug`, `updated_at`, `user_id`) values (2020-06-11 16:35:07, 1, Laravel, laravel, 2020-06-11 16:35:07, 1), (2020-06-11 16:35:07, 1, Laravel, laravel, 2020-06-11 16:35:07, 1), (2020-06-11 16:35:07, 1, Laravel, laravel, 2020-06-11 16:35:07, 1), (2020-06-11 16:35:07, 1, Laravel, laravel, 2020-06-11 16:35:07, 1), (2020-06-11 16:35:07, 1, Laravel, laravel, 2020-06-11 16:35:07, 1))

  at C:\xampp\htdocs\laravelblog\vendor\laravel\framework\src\Illuminate\Database\Connection.php:671
    667|         // If an exception occurs when attempting to run a query, we'll format the error
    668|         // message to include the bindings with SQL, which will make this exception a
    669|         // lot more helpful to the developer instead of just the database's errors.
    670|         catch (Exception $e) {
  > 671|             throw new QueryException(
    672|                 $query, $this->prepareBindings($bindings), $e
    673|             );
    674|         }
    675|

  1   C:\xampp\htdocs\laravelblog\vendor\laravel\framework\src\Illuminate\Database\Connection.php:464
      PDOException::("SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'Laravel' for key 'categories_name_unique'")

  2   C:\xampp\htdocs\laravelblog\vendor\laravel\framework\src\Illuminate\Database\Connection.php:464
      PDOStatement::execute()
